### PR TITLE
VIH-10227 Fix - Edit Hearing - new participants cannot log into Video Web

### PR DESCRIPTION
**Pull request by oliver-scott**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/VIH-10227

### Change description ###
Fixes an issue where if you edit a hearing and then add new participants, those participants are not added to any AAD groups and cannot log into Video Web.

This issue is specific to the Dev environment and is a side effect of the VPN issue. It messages Video Web and then assigns the participants to AAD groups, however on the Dev environment Video Web is unreachable as it outside VPN, so it throws an exception before it has assigned to AAD groups.

Fix by swapping these lines round - ie assign to groups before messaging Video Web. This also matches the order these events are done in `HearingReadyForVideoHandler`.

### CODE REVIEW

File: charts/vh-booking-queue-subscriber/values.dev.template.yaml [hunk 1]
```diff
@@ -2,12 +2,14 @@
 java:
   image: '${IMAGE_NAME}'
   releaseNameOverride: ${RELEASE_NAME}
+  environment:
```
Comment (oliver-scott) [inline charts/vh-booking-queue-subscriber/values.dev.template.yaml:5]: TODO revert these changes before merging

**Review by will-craig — APPROVED** [review]

**Review by komaldabhi1509 — APPROVED** [review]

**Review by komaldabhi1509 — APPROVED** [review]

**Review by will-craig — APPROVED** [review]
